components/create_comment/create_comment.jsx

```diff
@@ -724,7 +724,12 @@ export default class CreateComment extends React.PureComponent {
         let emojiPicker = null;
         if (this.props.enableEmojiPicker && !readOnlyChannel) {
             emojiPicker = (
-                <span className='emoji-picker__container'>
+                <span
+                    role='button'
+                    tabIndex='0'
+                    aria-label={Utils.localizeMessage('create_post.emojiPopup', 'Click to open emoji picker')}
```

Comment:
Should change to "Open emoji picker"

Reply:
Sorry, I missed that will change that too :)

Reply:
No worry, thanks @JayaKrishnaNamburu!